src/models/domain/model.spec.ts

```
@@ -1,15 +1,17 @@
-import { DataValue, Model } from './model';
+import { Model } from './model';
 import { User } from '../../users/domain/user';
 import { randomUUID } from 'crypto';
 import { Organization } from '../../organizations/domain/organization';
 import { ProductDataModel } from '../../product-data-model/domain/product.data.model';
+import { DataValue } from '../../passport/domain/passport';
+import { undefined } from 'zod';
```

Biome (1.9.4)

[error] Do not shadow the global "undefined" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)